### PR TITLE
[QA] SISRP-28467 - My Academics - Academic Records - Hover over text is in correct

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.berkeley.edu/browse/SISRP-28467

See DEV PR #6219

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/widgets/academics/academicRecordsController.js [hunk 1]
```diff
@@ -8,13 +8,13 @@ angular.module('calcentral.controllers').controller('AcademicRecordsController',
...
     postUrl: '',
-    postUrlHover: 'Request Transcript',
+    postUrlHover: 'Request your Official Transcript',
     isLoading: true,
     defaultRequestLink: 'http://registrar.berkeley.edu/academic-records/transcripts-diplomas'
   };
   $scope.lawTranscriptLink = {
     link: 'http://www.law.berkeley.edu/php-programs/registrar/forms/transcriptrequestform.php',
-    title: 'Request Law Transcript'
+    title: 'Request your official Law Transcript'
```
Comment (lyttam) [inline src/assets/javascripts/angular/controllers/widgets/academics/academicRecordsController.js]: @redconfetti the capitalization seems inconsistent - 'Official' is capitalized above but lowercase here.

**Comment by lyttam** [comment]
@redconfetti just curious, are these going to be moved to the link API eventually?

**Comment by redconfetti** [comment]
@lyttam - Not that I'm aware of. I know that our currently policy is that in most cases we are only migrating links to the CS Link API if they link to a PeopleSoft Classic or Fluid page, not external websites or systems. We have much work to do with the migration of existing links to the CS Link API, so static links like these will remain statically configured until a decision is made at a later date to migrate them.

**Comment by davidlimcheng** [comment]
@lyttam, I do agree we should make capitalization consistent - but for the sake of keeping this QA PR mirrored with the Master PR, I'm going to go ahead and merge this one in as well.

**Pull request merged**